server/types/schemas.ts

```diff
@@ -26,6 +26,7 @@ const VolunteerDataSchema = z
                 title: z.string(),
                 descriptionMarkdown: z.string(),
                 link: z.string(),
+                active: z.boolean(),
```

Comment:
can you take a look at `migrations.ts` and create a quick migration to automatically change the data on the staging site when this merges?